lib/wise_homex/models/account.ex

```diff
@@ -5,9 +5,9 @@ defmodule WiseHomex.Account do
 
   embedded_schema do
     belongs_to :owner, __MODULE__
-    has_many :tenancies, WiseHomex.Tenancy, foreign_key: :tenant_id
     has_many :admin_integrations, WiseHomex.AdminIntegration, foreign_key: :admin_id
     has_many :account_users, WiseHomex.AccountUser
+    has_one :tenancies, WiseHomex.Tenancy, foreign_key: :tenant_id
```

Comment:
This should be `:tenancy`

Reply:
Oops.. Fixed in https://github.com/wise-home/wise_homex/pull/216/commits/35c2daa1b757038a06207305b3584e04329d39ea